server/routes/viewModels/taskLists/caTasks.js

```diff
@@ -16,19 +17,18 @@ const createLicence = require('./tasks/createLicence')
 const finalChecks = require('./tasks/finalChecks')
 const caRereferDm = require('./tasks/caRereferDm')
 
-const namedTask = (name) => (/** @type {any} */ args = {}) => ({ task: name, ...args })
-
-const caBlocked = namedTask('caBlockedTask')
+const caBlocked = (errorCode) => () => ({ task: 'caBlockedTask', errorCode })
 const eligibilityTask = namedTask('eligibilityTask')
 const eligibilitySummaryTask = namedTask('eligibilitySummaryTask')
 const invisibleInformOffenderTask = namedTask('informOffenderTask')
```

Comment:
this can probably go - just need to check it's not having any weird visual side effects.